cloudformation/Prowler-Template.yaml

```diff
@@ -131,6 +131,7 @@ Parameters:
 Conditions:
   cSlackSecret: !Not [!Equals [ !Ref pSlackSecret, "NONE" ]]
   cDeployIngestLambda: !Not [!Equals [ !Ref pDomainEndpoint, "NONE" ]]
+  cDeployAlertLambda: !Not [!Equals [ !Ref pSlackSecret, "NONE" ]]
```

Comment:
See my comment in the other repo - The pSlackSecret will do two things that should be controlled independently.

Reply:
updated to use unique parameter